docusaurus/docs/operate/configs/relayminer_config.md

```diff
@@ -485,16 +486,44 @@ result in `Supplier` slashing if the `Proof` is required.
 - **Sufficient Balance**: Operators should regularly check their account balance
 and ensure they have enough funds to cover the submission fees for `Proofs`.
 
+The following command can be used to check the balance of a `Supplier` operator:
+```bash
+poktrolld query bank balance <supplier_operator_address> upokt
```

Comment:
#PUC w/ a --node to testnet 

Make it easy for people to copy-paste based on the most common use case